sector-builder-ffi/src/responses.rs

```diff
@@ -6,7 +6,29 @@ use drop_struct_macro_derive::DropStructMacro;
 use failure::Error;
 use ffi_toolkit::free_c_str;
 use libc;
-use sector_builder::{SectorBuilder, SectorBuilderErr, SectorManagerErr};
+use sector_builder::{SectorBuilder, SectorBuilderErr, SectorManagerErr, SectorStatus};
+
+#[repr(C)]
+#[derive(PartialEq, Debug)]
+pub enum FFISectorStatus {
+    Ok = 0,
+    ErrorNotSealed = 1,
```

Comment:
SUGGESTION: I would drop `ErrorNotSealed` and instead produce an error if somebody called `get_sealed_sector_status` with an id that was not the id of a sealed sector.

Reply:
sure